model/qti/CustomInteractionAsset/Extractor/BaseExtendedCustomInteractionAssetExtractor.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
This is the order of blocks as suggested by PSR-12

1. Opening <?php tag.
2. File-level docblock.
3. One or more declare statements.
4. ...

`declare()` therefore must be _under_ the file-level license block.

Reply:
agree but template here confused me a little bit 
https://oat-sa.atlassian.net/wiki/spaces/~369998401/pages/186221874/PhpStorm+live+templates

Reply:
Welp, you spot an issue with it :)

Reply:
Here's the common doc BTW
https://oat-sa.atlassian.net/wiki/spaces/NEX/pages/87361761/How+To+Insert+Open+Assessment+Technologies+SA+Licence+Header+Comment